v1/proto/service/gribi.proto

```diff
@@ -110,9 +110,15 @@ message ModifyRequest {
   // The network element selects the client with the highest
   // election_id as the primary. If the election_id sent by the
   // client matches the previous highest value, the newer client is
-  // conidered master, in order to allow for a client to reconnect
+  // considered master, in order to allow for a client to reconnect
   // without an external election having taken place.
   //
+  // Upon discovering a failover, the server:
+  //  * SHOULD stop processing pending AFTOperations that were sent by
+  //    the previous master.
```

Comment:
we've used `primary` instead of `master` in this file, do you mind changing it. There's also one occurence of `master on line 113 which needs to be changed.

Reply:
Good catch. Thanks. I'll file another PR to correct it since this one is already merged.